docker/bootstrap.sh

```diff
@@ -37,6 +37,18 @@ else
 
     mkdir -p /sytest
     tar -C /sytest --strip-components=1 -xf sytest.tar.gz
+
+    if [ -n "$PLUGIN" ]; then
+        mkdir /sytest/plugins
+        echo "--- Downloading plugins for sytest"
+        IFS=':'; for plugin in $PLUGIN; do
+            wget -q https://github.com/$plugin/archive/master.tar.gz -O plugin.tar.gz || {
```

Comment:
I'm not entirely convinced that tying this to github is the right thing to do. Can we make it more generic?

Reply:
It becomes difficult to select a seperator to pass multple plugins if we want to be able to pass whole URLs.
Something like # or : (with stripped https://) comes into my mind. But that would need good documentation.
Would are your thought on this?

Reply:
for the separator: can we just use space? PLUGINS="https://github.com/valkum/conduit_sytest/archive/master.tar.gz https://gitlab.com/project/plugin/-/archive/master/plugin-master.tar.gz"

for the directory: can we just make up a `mktemp`-style name? it doesn't need to be related to the URL of the plugin, does it?